plugin/src/main/resources/io/jenkins/plugins/analysis/core/portlets/PullRequestMonitoringPortlet/monitor.jelly

```diff
@@ -0,0 +1,21 @@
+<?jelly escape-by-default='true'?>
+
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler">
+
+  <st:adjunct includes="io.jenkins.plugins.echarts"/>
+
+  <j:choose>
+    <j:when test="${!it.hasIssues()}">
+      <div id="${it.id}" data="${it.getResultIssuesJsonModel()}"
+           style="width: ${it.preferredWidth}px; height: ${it.preferredHeight}px;"/>
+    </j:when>
+    <j:otherwise>
+      <h1>${%zeroIssues.title}</h1>
+      <p>${%zeroIssues.description}</p>
+    </j:otherwise>
+  </j:choose>
```

Comment:
The hasIssues check seems reversed to me, or maybe I just don't understand how j:when works.

Reply:
Ou of course, that's stupid..